src/features/events/utils/participantsFilterUtils.ts

```diff
@@ -0,0 +1,75 @@
+import Fuse from 'fuse.js';
+import {
+  ZetkinEventParticipant,
+  ZetkinEventResponse,
+} from 'utils/types/zetkin';
+
+type ListType = ZetkinEventParticipant | ZetkinEventResponse;
+
+//TODO filter cancelled list when API supports it
```

Comment:
This TODO seems irrelevant here, because there should not need to be any changes to this code when introducing cancelled participants.